tests/cylc-poll/14-intervals.t

```diff
@@ -33,18 +33,18 @@ suite_run_ok "${TEST_NAME_BASE}-run" \
 #-------------------------------------------------------------------------------
 TEST_NAME="${TEST_NAME_BASE}"
 LOG_FILE="${SUITE_RUN_DIR}/log/suite/log"
-grep_ok \
-    '\[t1\.1\] -health check settings: submission.*, polling intervals=PT2S,6\*PT10S,...' \
-    "${LOG_FILE}"
-grep_ok \
-    '\[t2\.1\] -health check settings: submission.*, polling intervals=PT2S,6\*PT10S,...' \
-    "${LOG_FILE}"
-grep_ok \
-    '\[t1\.1\] -health check settings: execution.*, polling intervals=2\*PT1S,10\*PT6S,...' \
-    "${LOG_FILE}"
-grep_ok \
-    '\[t2\.1\] -health check settings: execution.*, polling intervals=2\*PT1S,10\*PT6S,...' \
-    "${LOG_FILE}"
+
+PRE_MSG='-health check settings:'
+for INDEX in 1 2; do
+    for STAGE in 'submission' 'execution'; do
+        POLL_INT='PT2S,6\*PT10S,'
+        if [ "${STAGE}" == 'execution' ]; then
```

Comment:
As discussed, please use `[[ ... ]]` instead of `[ ... ]` in `if` blocks.

Reply:
Sure, thank you for explaining the difference to me. Preference noted for future work.